README

```diff
@@ -149,6 +149,7 @@ REQUIREMENTS:
         libpython (optional)
         libidn2 or libidn (optional)
         elfutils >= 158 (optional)
+        polkit >= 0.113-22-gc78819245f (optional)
```

Comment:
With your follow-up commit, this can probably be dropped again? 
Or will ```i18n.merge_file``` use the system installed polkit.(its|loc) file if available?

Reply:
Yeah, update to drop the version.